CustomScript/HandlerManifest.json

```diff
@@ -1,13 +1,17 @@
-[{
-  "name": "CustomScript",
-  "version": "1.5.1.0",
-  "handlerManifest": {
-    "installCommand": "customscript.py -install",
-    "uninstallCommand": "customscript.py -uninstall",
-    "updateCommand": "customscript.py -update",
-    "enableCommand": "customscript.py -enable",
-    "disableCommand": "customscript.py -disable",
-    "rebootAfterInstall": false,
-    "reportHeartbeat": false
+[
+  {
+    "namespace": "Microsoft.OSTCExtensions",
+    "name": "CustomScriptForLinux",
+    "extension_version": "1.5.2.0",
+    "version": 1,
```

Comment:
@boumenot unless you dynamically need to update this document (you should not) I see you already ran jq once so it should be fine. I'd rather rely on the value covered by the spec than losing the ability to use jq.